lib/server.c

```diff
@@ -87,7 +87,7 @@ int fuzi_q_server(fuzi_q_mode_enum fuzz_mode, picoquic_quic_config_t* config, ui
     }
 #endif
     if (ret == 0) {
-        fuzi_q_ctx.quic = picoquic_create_and_configure(config, picoquic_demo_server_callback, &picoquic_file_param, current_time, NULL);
+        fuzi_q_ctx.quic = picoquic_create_and_configure(config, picoquic_h09_server_callback, &picoquic_file_param, current_time, NULL);
```

Comment:
The change in `picoquic_set_alpn_select_fn` make this function unnecessary. The callback will be set automatically as a function of the ALPN. Setting this to NULL is better than forcing H09, because it allows fuzz testing of either H09 or H3.

```suggestion
fuzi_q_ctx.quic = picoquic_create_and_configure(config, NULL, &picoquic_file_param, current_time, NULL);
```